conftest.py

```diff
@@ -0,0 +1,37 @@
+"""
+Configure pytest before running any test.
+"""
+import sys
+
+import mock
+
+
+def pytest_configure(config):
```

Comment:
Mocking the third party modules here (currently only `student` module) since these modules are imported at top level of files and thus are imported even before the tests are run but since there is no such module in the app, app is crashed even before starting the tests.

Therefore we need to mock these even before the tests are run. `conftest.py` does that.

Reply:
Update:
Resolve the signals issue. and moved to code for mocking the module into `tests/__init__.py`